### PR TITLE
Add stats feature (partial)

### PR DESCRIPTION
**Pull request by Zhi-You**
Part of #112. Requires polishing up before it can be closed. Current implementation leeches on 'about' command to print out.
Therefore, the function has to be updated in menuParser and Commands.

Requires #111 to store user's progress in Eduke8 so that their stats can be shown.

Will eventually cover #96.

### CODE REVIEW

File: src/main/java/seedu/eduke8/stats/StatsCalculator.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    public StatsCalculator() {
+        topicsStorage = new TopicsStorage(DATA_PATH);
+
+        try {
+            topicList = new TopicList(topicsStorage.load());
+        } catch (ParseException | IOException e) {
+            new Ui().printError(ERROR_STORAGE_FAIL);
+            LOGGER.log(Level.WARNING, "Error reading or writing local files.");
+        }
```
Comment (kstonekuan) [inline src/main/java/seedu/eduke8/stats/StatsCalculator.java]: I know this is a draft but you should probably be taking the `TopicList` in the constructor

Comment (Zhi-You) [reply src/main/java/seedu/eduke8/stats/StatsCalculator.java]: Yep, thanks! It is now updated as such in the new pull request.

File: src/main/java/seedu/eduke8/stats/Stats.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+import seedu.eduke8.common.Displayable;
+import seedu.eduke8.topic.Topic;
+import seedu.eduke8.topic.TopicList;
+import seedu.eduke8.ui.Ui;
+
+import java.util.ArrayList;
+
+
+public class Stats {
```
Comment (kstonekuan) [inline src/main/java/seedu/eduke8/stats/Stats.java:11]: Nice, good implementation of this feature

**Review by kstonekuan — APPROVED** [review]
LGTM

**Pull request merged**